.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     - stage: Build and Verify
       script:
         - make .gitvalidation
-        - make gofmt
+        - make fmt
```

Comment:
I think we will need to install clang-format in travis for this to work.

Reply:
> I think we will need to install clang-format in travis for this to work.

That [doesn't seem to work][1], although [clang-format is packaged for 14.04][2].  Any thoughts on resolving that?

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/357022001#L555
[2]: https://packages.ubuntu.com/search?suite=trusty&searchon=names&keywords=clang-format

Reply:
Ah, setting an explicit major/minor [worked][1].